apph.adoc

```diff
@@ -117,15 +114,14 @@ In the former case, listing the time variable in the **`coordinates`** attribute
    attributes:
        :featureType = "timeSeries";
 ----
-In this example, **`humidity(i,o)`** is element o of time series i, and associated with the coordinate values **`time(o)`**, **`lat(i)`**, and **`lon(i)`**.
-Either the instance (station) or the element (time) dimension may optionally be the netCDF unlimited dimension.
+In this example, `humidity(i,o)` is element `o` of time series `i`, and associated with the coordinate values `time(o)`, `lat(i)`, and `lon(i)`.
```

Comment:
Where are we on Oxford commas? Or is that taking the style guide a bit too far? :smile:

Reply:
Not too far for me :) I would support the use of Oxford commas.

Reply:
Now you say this, I feel I can share that it would be nice, in my perfectionistic eyes, to be consistent across the document and either go all (relevant cases) with, or all without. Though I don't mind which one, of the two. Is such a thing included in our new style guide, and if not could we include our own addition to advise that one should or (whatever we chose as the best option) should not be used?